src/integrationTest/java/uk/gov/hmcts/darts/hearings/controller/HearingsControllerGetTranscriptsTest.java

```diff
@@ -108,6 +109,20 @@ void hearingsGetTranscriptEndpointTwoObjectsReturned() throws Exception {
         JSONAssert.assertEquals(expected, actualResponse, JSONCompareMode.NON_EXTENSIBLE);
     }
 
+    @Test
+    void hearingsGetTranscriptEndpointTranscriptWithHiddenDocumentNotReturned() throws Exception {
+        HearingEntity hearingEntity = dartsDatabase.getHearingRepository().findAll().get(0);
```

Comment:
Do we need to find all here could we not grab any? (Thinking about test performance)

Reply:
There's only 1 in the DB at this point so it doesn't do any harm